core/src/main/java/com/netflix/conductor/service/ExecutionService.java

```diff
@@ -174,20 +175,18 @@ public Task getPendingTaskForWorkflow(String taskReferenceName, String workflowI
 		return executor.getPendingTaskByWorkflow(taskReferenceName, workflowId);
 	}
 
-	public boolean ackTaskRecieved(String taskId, String consumerId) throws Exception {
-		Task task = getTask(taskId);
-
-		if (task != null) {
-			String queueName = QueueUtils.getQueueName(task);
-			if(task.getResponseTimeoutSeconds() > 0) {
```

Comment:
Essentially this logic is removed, context on why we can remove it?

Reply:
@cyzhao , the un-acked queue mechanism was used to re-queue the message in case of `ResponseTimeoutSeconds`. Unacked queue is only intended be only used to when the task is given out to the worker and worker acks back saying it has received and processing work.

Now since the getter of responeTimeoutSeconds has been fixed to return the actual value instead of 0, the Sweeper will evaluate and reschedule the task by requeueing it.